core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosCoarseGrainedSchedulerBackend.scala

```diff
@@ -103,6 +103,7 @@ private[spark] class MesosCoarseGrainedSchedulerBackend(
   private val stateLock = new ReentrantLock
 
   val extraCoresPerExecutor = conf.getInt("spark.mesos.extra.cores", 0)
+  val maxGpus = conf.getInt("spark.mesos.gpus.max", 0)
```

Comment:
I'm not saying it's not sensible. I'm just trying to figure what I can do to tell it to accept all GPUs in the offer (which is what I want in my setup). Some offers have more than others and it feels weird to just pick a really big number to ensure that I get them all.